recipes/atomsk/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "atomsk" %}
+{% set version = "0.10.1" %}
+{% set sha256 = "28dac7c83a7d0b76b773eb37d921300fef8120faa829d47eae133d8e2889b22b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pierrehirel/{{ name }}/archive/Beta-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - gcc
+    - make 
+    - lapack
+  host:
+    - lapack
+    - libgcc
+  run:
+    - lapack
+    - libgcc
```

Comment:
after you add the syntax for conda build 3, could you also remove the ``libgcc`` from ``run`` and ``host`` ?

Reply:
We need to add a new lint for `gcc`/`libgcc` that points to the docs on how to use the compiler jinja syntax.